### PR TITLE
Correct a lot of the typos

### PR DESCRIPTION
**Pull request by Brandon-Huu**
## About the PR

Removed multiple instances of programmer spelling

## Why / Balance

Some of the typos were user facing and made us look bad. stupid dog.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
NOOO

**Comment by Ilya246** [comment]
this is a massive breaking change and will break most downstreams

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Datafields should be using the string-less version, rest looked okay. The only actual change downstreams should need to do is the ambient music update for interruptible if they have their own and git should handle the rest.

File: Content.Client/CloningConsole/UI/CloningConsoleWindow.xaml.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void Populate(CloningConsoleBoundUserInterfaceState state)
                         case ClonerStatus.ScannerOccupantAlive:
                             CloningActivity.Text = (Loc.GetString("cloning-console-component-msg-scanner-occupant-alive"));
                             break;
-                        case ClonerStatus.OccupantMetaphyiscal:
+                        case ClonerStatus.OccupantMetaPhysical:
```
Comment (Ilya246) [inline Content.Client/CloningConsole/UI/CloningConsoleWindow.xaml.cs:56]: "metaphysical" is one word, no?

**Comment by Brandon-Huu** [comment]
wtf

**Pull request closed without merging**